includes/namespace.php

```diff
@@ -157,13 +158,16 @@ function remove_site( $error, $site ) {
 /**
  * Handles plugin deactivation.
  *
+ * @SuppressWarnings(PHPMD.BooleanArgumentFlag)
```

Comment:
Surprised this wasn't an issue before

Reply:
I added the default in this PR. It should have always been there.